ui/app/AppLayouts/Chat/views/communities/CommunityPermissionsView.qml

```diff
@@ -0,0 +1,63 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+
+import SortFilterProxyModel 0.2
+import utils 1.0
+
+import "../../../Chat/controls/community"
+
+Flickable {
```

Comment:
Ok, I will keep it like this to have consistency between `Permission` views since the other are also `Flickable` but will create a task to update all them together.

Here the task: https://github.com/status-im/status-desktop/issues/8584

Reply:
Why not do it here? :) It's new code, there's no rush since it's all mocked anyway, and StatusScrollView is a Flickable too